h2o-persist-gcs/src/test/java/water/persist/PersistGcsTest.java

```diff
@@ -0,0 +1,63 @@
+package water.persist;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.*;
+import water.fvec.Chunk;
+import water.fvec.FileVec;
+import water.fvec.Frame;
+import water.util.FileUtils;
+
+import java.net.URI;
+import java.util.Arrays;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class PersistGcsTest extends TestUtil {
+  @BeforeClass
+  public static void setup() {
+    stall_till_cloudsize(5);
```

Comment:
There is no need for a cloud size of 5. Let's change it to 1.

Reply:
In AWS connector you use 5.

Reply:
Resolved in new PR https://github.com/h2oai/h2o-3/pull/2119